Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/full/json/JsonBufferedFileWriter.java

```diff
@@ -55,8 +57,5 @@ public void writeRecords(Batch batch) throws Exception {
     public void close() throws Exception {
         bw.write("]");
         bw.close();
-
```

Comment:
Just to validate, if I understood correctly, this part is no longer required because it's executed in the `afterJob` in the FileUpdater?

Reply:
yes basically FileUpdater replaced this functionallity, problem was that when the writter is executed even is the job only process a single submission because an error, oppositive to this Updated only get executed at the end of the full job executed.